db/table_cache.cc

```diff
@@ -283,19 +285,19 @@ Status TableCache::Get(const ReadOptions& options,
                        GetContext* get_context, HistogramImpl* file_read_hist,
                        bool skip_filters, int level) {
   Status s;
+  Cache::Handle* handle = nullptr;
   if (get_context->range_del_agg() != nullptr &&
       !options.ignore_range_deletions) {
```

Comment:
@ajkr for now I think it is safer to make the default to be true for now. In case we find another performance regression somewhere else, it will be complicated to further patch 5.0.